README.md

```diff
@@ -1,76 +1,76 @@
 # \*kali
 
-\*kali is a calendar application for berlin-based women*-who-code groups (code curious, heart of code, pyladies,...).
-Through \*kali, all events of all groups can be seen in one calender view (in order to...build a stronger/bigger community??). Every group can create, read, update and delete events that can be accessed (through a website?) by users who are interested in these events.
+*This is an open source project that was started in 2019 by the [code curious](https://www.codecurious.org) study group "Ruby Stars".* 
+
+\*kali is a calendar application for berlin-based groups of women* who learn how to code. As a platform it gathers all events of those groups, in order to build a stronger community and increase their visibility. Every group can create an account and post their events so that everyone interested can browse through them.
+
+...At least, thats the plan :) Under <b>First Mockups</b> and <b>Minimum Viable Product</b> below you can see how we imagine the calender to be. For that, we need your help. If you want to contribute to our project, go for it and check out our [guidelines](CONTRIBUTORS.md).
```

Comment:
Since `First Mockups` and `Minimum Viable Product` are headers we could link them to the actual section of the readme. This is how to write it:
`[First Mockups](#first-mockups)` 
`[Minimum Viable Product](#minimum-viable-product)`